src/test/java/com/kodilla/ecommercee/domain/OrderEntityTestSuit.java

```diff
@@ -0,0 +1,116 @@
+package com.kodilla.ecommercee.domain;
+
+import com.kodilla.ecommercee.exception.OrderNotFoundException;
+import com.kodilla.ecommercee.service.CartService;
+import com.kodilla.ecommercee.service.OrderService;
+import com.kodilla.ecommercee.service.UserService;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import javax.transaction.Transactional;
+
+import java.util.List;
+
+import static org.junit.Assert.*;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+public class OrderEntityTestSuit {
+    @Autowired
+    OrderService orderService;
+    @Autowired
+    UserService userService;
+    @Autowired
+    CartService cartService;
+
+    @Transactional
+    @Test
+    public void testFindAll() {
+        //Given
+        int ordersTableSizeBeforeTest = orderService.getOrderList().size();
+
+        //When
+        orderService.addOrder(new Order());
+        orderService.addOrder(new Order());
+        orderService.addOrder(new Order());
+
+        //Then
+        assertEquals(3, orderService.getOrderList().size() - ordersTableSizeBeforeTest);
+    }
+
+    @Transactional
+    @Test(expected = OrderNotFoundException.class)
+    public void testOrderNotFoundException() throws OrderNotFoundException {
+        //Given
+        Order order = orderService.addOrder(new Order());
+
+        //When
+        orderService.deleteOrder(order);
+        orderService.getOrderById(order.getId());
+    }
+
+    @Transactional
+    @Test
+    public void testDeleteOrder() {
+        //Given
+        Order order = orderService.addOrder(new Order());
+        int ordersSizeBeforeDelete = orderService.getOrderList().size();
+
+        //When
+        orderService.deleteOrder(order);
+
+        //Then
+        assertEquals(ordersSizeBeforeDelete - 1, orderService.getOrderList().size());
+    }
+
+    @Transactional
+    @Test
+    public void testFindById() throws OrderNotFoundException{
+        //Given
+        Order order = orderService.addOrder(new Order());
+
+        //When
+        Order testOrder = orderService.getOrderById(order.getId());
+
+        //Then
+        assertEquals(order, testOrder);
+    }
+
+    @Transactional
+    @Test
+    public void testAdd() {
+        //Given
+        Order order = orderService.addOrder(new Order());
+
+        //When
+        List<Order> orders = orderService.getOrderList();
+
+        //Then
+        assertTrue(orders.contains(order));
```

Comment:
to jest tożsame z testem testFindAll(). Testujesz dokładnie to samo tylko w inny sposób. Tam dodajesz 3 zamówienia poprzez addOrder() i sprawdzasz czy rozmiar się zmienił. Tutaj dodajesz 1 zamówienie i sprawdzasz czy pojawiło się na liście.